department-of-veterans-affairs/va.gov-cms
# VACMS-2485: Add lists of lists to benefit main paragraph.

## Description
See #2485 

## Testing done
Behat

## QA steps
As an admin, go to `admin/structure/types/manage/health_care_region_detail_page/fields/node.health_care_region_detail_page.field_content_block` and visually verify that "Lists of lists" is selected.
